### PR TITLE
Info on connect point between host and docker filesystems

### PR DESCRIPTION
**Pull request by jonkiparsky**
PR because development is protected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,15 @@ on which the Django project is running (service "web"), run `make bash-shell`.
 * In order to access the Django shell, run `make django-shell`.
 * In order to access the mysql shell, run `make db-shell`.
 
+### Writing to/reading from the host machine
+If you need to write out data from a local impact-api instance to your host
+machine's drive, the directory `/wwwroot` on the docker 
+container is mapped to `impact-api/web/impact` on your host machine. If you are
+working in a django shell, creating a filehandle `f = open("/wwwroot/foo.txt", "w")`
```
Comment (pbx) [inline README.md]: `s/django shell/Django shell in the container/`

File: README.md [hunk 2]
```diff
@@ -98,6 +98,15 @@ on which the Django project is running (service "web"), run `make bash-shell`.
...
+will allow you to write out data to your host machine.
```
Comment (pbx) [inline README.md]: `s/to your/that can also be seen from your/`

**Review by pbx — CHANGES_REQUESTED** [review]
Thanks for adding this. A couple small clarification suggestions.

File: README.md (hunk 2 above)
Comment (jonkiparsky) [reply README.md]: @pbx back to you

**Review by pbx — APPROVED** [review]

**Pull request merged**